src/Assets/Javascript/locations-map.js

```diff
@@ -23,50 +122,60 @@ const initLocationsMap = () => {
     },
     zoom: 11,
     center: {
-      lat: locations[0].lat,
-      lng: locations[0].lng
-    }
-  })
+      lat: centerLat,
+      lng: centerLng
+    },
+    styles: [
+      {
+        featureType: "poi.business",
+        stylers: [
+          {
+            visibility: "off"
+          }
+        ]
+      },
+      {
+        featureType: "poi.park",
+        elementType: "labels.text",
+        stylers: [
+          {
+            visibility: "off"
+          }
+        ]
+      }
+    ]
+  });
 
-  google.maps.event.addListener(map, 'click', function () {
-    infoWindow.close()
-  })
+  const locations = window.trainingLocations;
 
   for (let i = 0, length = locations.length; i < length; i++) {
-    const location = locations[i]
-    const latLng = new google.maps.LatLng(location.lat, location.lng)
-
-    const marker = new google.maps.Marker({
-      position: latLng,
-      map,
-      title: location.name,
-      icon: locationMarker,
-      zIndex: 2
-    });
+    const location = locations[i];
+    const latLng = new google.maps.LatLng(location.lat, location.lng);
+
+    const closedContent = document.createElement("div");
+    closedContent.innerHTML = location.name;
 
-    (((marker, location) => {
-      google.maps.event.addListener(marker, 'click', e => {
-        const windowsContent = `
-          <div class="search-info-window">
-            <h3 class="govuk-heading-s">${location.name}</h3>
-            ${ location.address ? `<p class="govuk-body">Address: ${location.address}</p>` : '' }
-          </div>
-        `
-        infoWindow.setContent(windowsContent)
-        infoWindow.setOptions({maxWidth:250})
-        infoWindow.open(map, marker)
-      })
-    }))(marker, location)
-
-    bounds.extend(latLng)
-    markers.push(marker)
+    const openContent = document.createElement("div");
+    openContent.innerHTML = `
+      ${`<h3 class="govuk-heading-s">${location.name}</h3>`}
+      ${location.address ? `<p class="govuk-body">Address:<br> ${location.address}</p>` : ""}
+      <button class="popup-bubble__close">&times;<span class="govuk-visually-hidden">Close this popup</span></button>
+    `;
+
+    const popup = new Popup(latLng, closedContent, openContent);
+    popup.setMap(map);
+
+    // Extend the bounds by the first 5 locations so we get a decent number as part of the first view.
+    if (i < 5) {
+      bounds.extend(latLng);
+    }
   }
 
   // Use provider address to center and zoom when only one location
   if (locations.length > 1) {
-    map.fitBounds(bounds)
-    map.panToBounds(bounds)
+    map.fitBounds(bounds);
+    map.panTo(new google.maps.LatLng(centerLat, centerLng));
```

Comment:
Shouldn't we still be panning to bounds here? The logic for zoom level of the map on this page shouldn't change.